### PR TITLE
Split branches builds in to separate test suites

### PR DESCRIPTION
**Pull request by jystic**
Should speed up builds a bit, running tests in parallel.

! @amosr @tranma

### CODE REVIEW

**Comment by tranma** [comment]
yes please
💯

**Comment by jystic** [comment]
Not sure if these are the right splits, if we notice one build taking longer than others would be good to split again

**Comment by jystic** [comment]
Worth also noting that I didn't bother splitting the master build, that one runs all the tests before publishing a new binary

File: icicle-compiler/test/test.hs [hunk 1]
```diff
@@ -41,52 +38,104 @@ import qualified Icicle.Test.Sea.Text
...
+      Icicle.Test.Sea.Psv.tests
+    , Icicle.Test.Sea.Psv.Schema.tests
+    , Icicle.Test.Sea.PsvFission.tests
+    ]
+
+zebra :: TestSuite
+zebra =
+  TestSuite "zebra" [
+      Icicle.Test.Sea.Zebra.tests
+    , Icicle.Test.Sea.Seaworthy.tests
```
Comment (tranma) [inline icicle-compiler/test/test.hs]: this is psv

File: icicle-compiler/test/test.hs [hunk 2]
```diff
@@ -41,52 +38,104 @@ import qualified Icicle.Test.Sea.Text
...
+    , Icicle.Test.Sea.Name.tests
```
Comment (tranma) [inline icicle-compiler/test/test.hs]: this is not zebra specific

**Pull request merged**